exercises/J-array-get-set/exercises2.js

```diff
@@ -6,8 +6,8 @@
   - change the first value in the array to the number 1
 */
 
-var numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+const numbers = [1, 2, 3]; // Don't change this array literal declaration
+numbers.push(4);
```

Comment:
Yep good Musa! Can also do `numbers[3] = 4` as well if you were curious about another solution